docs/explanations/pro_token_and_machine_usage.rst

```diff
@@ -0,0 +1,65 @@
+.. _pro_token_and_machine_usage:
+
+Machine counts and Token usage for Ubuntu Pro
+*********************************************
+
+Pro Tokens:
+-----------
+
+- Obtaining Tokens:
+    After subscribing to Ubuntu Pro, you can access your tokens through the Ubuntu Pro dashboard. Each token can be used to attach multiple machines depending on the subscription.
+
+- Attaching Tokens:
+    Command: Use ``sudo pro attach <token>`` to connect a machine to your Ubuntu Pro subscription.
+
+- Detaching Tokens:
+    Command: Use ``sudo pro detach`` to remove a machine from the subscription.
+
+
+Monitoring Active Machines:
+---------------------------
+
+- Machine Limits: 
+    Each subscription type has a limit on the number of machines it can cover. For personal subscriptions, this limit is 5 physical machines with unlimited VMs on these machines. For enterprise subscriptions, the limit is based on the number of physical hosts and VMs.
+
+- Active Machine Tracking: 
+    The Ubuntu Pro dashboard and tools like Landscape help track the number of active machines, ensuring compliance with subscription limits.
+
+
+- Unused Licenses and Active Users:
+    The Ubuntu Pro dashboard provides a comprehensive view of all active and attached machines. It shows the number of active machines and helps track unused licenses, ensuring that you remain within the subscription limits.
+
+- Monitoring with Landscape:
+    Setup: Install the Landscape client and connect it to the Landscape server.
+
+    Audit Machines: Use Landscape’s remote script execution capability to run audit scripts, such as ``uastatus.sh``, to gather detailed information about your Ubuntu Pro subscription across all machines. This information can be exported into a CSV report for further analysis.
+
+Dealing with Multiple VMs:
+---------------------------
+- VM License Management: 
+    In an IaaS environment where node control is limited, you can contact Canonical's sales team to discuss purchasing licenses per single VM rather than per physical node.
+
+- Centralized Management: 
+    For enterprise environments, using Landscape ensures all Ubuntu instances have the necessary Ubuntu Pro entitlements, helping identify any discrepancies or unauthorized usage.
+
+Handling Overage of Active Machines:
+------------------------------------
+Canonical monitors the number of active machines. Occasional overages are generally acceptable, but persistent overages may lead to subscription termination.
+
+Frequently Asked Questions:
+===========================
+
+- Can I use Ubuntu Pro Desktop for server use cases?
+    No, using Ubuntu Pro Desktop for non-desktop use cases violates the terms of service. Use an Ubuntu Pro Server subscription for server use.
+
+- What if I need more than 5 machines for personal use?
```

Comment:
I think this can be combined with the point about centralised management/Landscape so that it's not so much an FAQ but rather a point about small-scale vs large-scale use so that it fits better with the content on the page